src/coreclr/pal/src/libunwind/include/libunwind-arm.h

```diff
@@ -277,9 +277,9 @@ unw_tdep_context_t;
   unw_tdep_context_t *unw_ctx = (uc);                                   \
   register unsigned long *unw_base __asm__ ("r0") = unw_ctx->regs;      \
   __asm__ __volatile__ (                                                \
-    "stmia %[base], {r0-r15}"                                           \
-    "adds %[base], #64"                                                 \
-    "vstmia %[base], {d0-d15}"                                          \
+    "stmia %[base], {r0-r15}\n"                                           \
+    "adds %[base], #64\n"                                                 \
+    "vstmia %[base], {d0-d15}\n"                                          \
```

Comment:
Was it breaking the compilation without this change? We usually don't change the vendor's code unless strictly necessary. The proper way is to upstream the changes which @janvorli was planning to do.

Reply:
Yes. The non-thumb code is faulty and doesn't compile.

Reply:
Faulty as in just missing \n?
we have significantly changed it for the next version, see https://github.com/dotnet/runtime/pull/62092.

Reply:
Just missing \n - without the \n's the inline assembly all goes on one line, and that ain't valid at all

Reply:
ok, then we are in luck, i already added the missing `\n`s there, in addition to upstream fixes and our floating point patch: https://github.com/dotnet/runtime/pull/62092/commits/231b4568059c6a6bf2819d50b99883b666de6fe7